src/context/logger/loggerContextController/LoggerContextController.tsx

```diff
@@ -0,0 +1,20 @@
+import { useMemo, useState } from 'react';
+
+import { LoggerContext } from '../loggerContext/LoggerContext';
+import { loggerService } from '../loggerService/LoggerService';
+import { LoggerContextValueType } from '../loggerContext/LoggerContext.types';
+
+import { LoggerContextControllerProps } from './LoggerContextController.types';
+
+export const LoggerContextController = ({ children }: LoggerContextControllerProps) => {
+  const [state] = useState(loggerService.logger);
+
+  const value: LoggerContextValueType = useMemo(
```

Comment:
Can we pass the state directly as a value?

Reply:
Sure :). Done.